packages/kyt-core/utils/jest/preprocessor.js

```diff
@@ -1,17 +1,57 @@
-const mergeAll = require('ramda').mergeAll;
-const babel = require('../../config/babel')();
 const babelJest = require('babel-jest');
-const buildConfigs = require('../buildConfigs');
-const config = require('../kytConfig')();
+const fs = require('fs');
+const shell = require('shelljs');
+const { userBabelrcPath, userRootPath } = require('kyt-utils/paths')();
+const logger = require('kyt-utils/logger');
+const resolve = require('resolve');
 
-const { clientConfig } = buildConfigs(config);
+/**
+ * Support preset and plugin definitions in .babelrc
+ * without the babel-* prefixes.
+ *
+ * E.g. {presets: ['babel-preset-react']} --> {presets: ['react']}
+ *
+ * @param  {String} prefix One of either 'babel-plugin' or 'babel-preset'
+ * @param  {String} dep    The dependency
+ * @return {String}        The normalized dependency
+ */
+const normalizeDep = (prefix, dep) => {
+  let resolved;
+  if (dep.indexOf('babel') !== 0) {
+    try {
+      resolved = resolve.sync(`${prefix}-${dep}`, { basedir: userRootPath });
+    } catch (e) { /* eslint-disable no-empty */ }
```

Comment:
I think so right? Because this would be an exceptional case (where there's a babel plugin referenced that doesn't exist so it should fail loudly, I think.